tests/parallel/treadafterwrite.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   errormsg: "'foo' not disjoint from 'foo'"
   line: 23
-  disabled: "true"
+    disabled: "true"
```

Comment:
Created disabled around 2014-12 in a commit that references #1597

Reply:
Does it work?

If not, then delete.

Reply:
For completeness: 

```
FAIL: tests/parallel/treadafterwrite.nim c                         ( 3.22 sec)
Test "tests\parallel\treadafterwrite.nim" in category "parallel"
Failure: reMsgsDiffer
Expected:
'foo' not disjoint from 'foo'

Gotten:



```
Yes, the 'gotten' field  is blank.